src/components/drawer/Swipeable.tsx

```diff
@@ -11,10 +11,12 @@ import React, {Component} from 'react';
 import {Animated, StyleSheet, View, I18nManager} from 'react-native';
 import {PanGestureHandler, TapGestureHandler, State} from 'react-native-gesture-handler';
 import {Constants} from '../../helpers';
+import {HapticService, HapticType} from '../../services';
 
 
 const DRAG_TOSS = 0.05;
 const LEFT_TOGGLE_THRESHOLD = 0.6;
+const HAPTIC_METHOD = HapticType.impactMedium;
```

Comment:
Is this working? The other PR merged?

Reply:
It is not merged yet - I added a link in the description.
And yes it works, I checked it..

Reply:
If you're using HAPTIC_METHOD only once, why store it in a const?

Reply:
Also, why `impactMedium` is the default?

Reply:
These are the UX guidelines.
We can open it to a discussion in the future if any user will ask to change it.

Reply:
I'm asking because our private config (Drawer.config) is setting it to `impactLight`, so it should be the default or it should be open for user's change.

Reply:
It has updated to `impactMedium`, which is the default now.